LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, Tyler Roscoe
```

Comment:
I have noticed some repositories in the GPII org have this added to them:

```
All modules are Copyright 2014 Raising the Floor - International except
where noted otherwise in the code itself, or if the modules reside in a
separate directory, they may contain explicit declarations of copyright
in both the LICENSE file in the directory in which they reside and in the
code itself. No external contributions are allowed under licenses which are
fundamentally incompatible with the BSD or Apache licenses that the GPII is
distributed under.
```

Reply:
Do we have official guidance on Licenses?

If so, point me to it and I'm happy to use it.

If not, I'd rather stick with the generic license until we have a formal decision.

Reply:
https://wiki.gpii.net/w/Licensing

The copyright should be attributed to your  organization—in this case, Raising the Floor U.S. If you make substantive changes to a file, you can add your organization to the copyright header.

Reply:
Thanks Colin. I updated LICENSE (now LICENSE.txt) per those instructions.